app/src/pages/CountsPage/container/CounterPageContainer.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import Counter from '../components/Counter';
+
+class CounterPageContainer extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            countValue: 0,
+            parityType: 'even/odd',
+        };
+    }
+    handlIncrement = () => {
+        this.setState((state) => {
+            const countValue = state.countValue + 1;
+            if (countValue % 2 == 0) {
+                const parityType = 'even';
+                const color = 'red';
+                return {
+                    ...state,
+                    countValue,
+                    parityType,
+                    color,
+                };
+            } else {
+                const parityType = "odd";
+                const color = 'green';
+                return {
+                    ...state,
+                    countValue,
+                    parityType,
+                    color,
+                }
+            }
+        });
```

Comment:
try to think how you can improve your solution and make only one return statement